Gems/Atom/Feature/Common/Code/Source/Material/MaterialConverterSystemComponent.cpp

```diff
@@ -183,6 +187,13 @@ namespace AZ
                 sourceData.SetPropertyValue(Name{"roughness.factor"}, roughness.value());
             }
 
+            if (!materialData.GetRoughnessFactor().has_value() && abs(materialData.GetShininess()) > AZ::Constants::FloatEpsilon)
```

Comment:
Whats the abs needed for? Is a negative value valid/expected for shininess?

Reply:
changed.